app/routes/discover.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+    model() {
+        return Ember.RSVP.hash({
+//            preprints: this.store.findAll('preprint'),
```

Comment:
In general we try to avoid returning a hash from the model hook.  Helpers like `link-to` expect to be given a single model, and the page ends up not rendering correctly.